CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ Remove the `s3_access_key` and `s3_secret_key` variables from your `orca.tf` fil
 ### Added
 
 - *ORCA-845* - Created IAM role for RDS S3 import needed for Aurora v2 upgrade.
+- *ORCA-792* - Added DB comparison script for the temporary EC2 to compare databases post migration.
```

Comment:
I added the directory to the changelog, we can put it in the bin directory but I just put it in the EC2 folder since the EC2 will be using the script and installed on the instance when it is created to keep it separated.

And [ORCA-868](https://bugs.earthdata.nasa.gov/browse/ORCA-868) will provide the instructions with deployment of the instance as well as script usage.

Reply:
That's kind of what I was initially thinking since that's where the DR bucket modules are at and it will be similar to those where it's deployed on it's own that way it can be created/destroyed without affecting ORCA. Unless do you think we should make a new directory for it since it's not really apart of ORCA?

Reply:
Since this is only for testing the DB after migration, it will be a manual deployment step that we will ask users to run. So we can either put them under modules/db_compare_instance/ just like dr_buckets or create a new dir. Anything is fine with me.